fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -842,23 +842,22 @@ std::set<Tag> CommitBatchContext::getWrittenTagsPreResolution() {
 					transactionTags.insert(cacheTag);
 				}
 			} else if (m.type == MutationRef::ClearRange) {
-				KeyRangeRef clearRange(KeyRangeRef(m.param1, m.param2));
-				auto ranges = pProxyCommitData->keyInfo.intersectingRanges(clearRange);
```

Comment:
So, the `keyInfo.intersectingRanges` is heavier than the manual iteration, therefore we want to get rid of it. Do I understand correctly? Thanks

Reply:
Yes. `intersectingRanges` does two key lookups, for range `begin` and `end`. `rangeContaining` only does one lookup.